vspc/server.py

```diff
@@ -143,6 +150,11 @@ async def handle_vmotion_begin(self, writer, data, vm_uuid):
         writer.write(IAC + SB + VMWARE_EXT + VMOTION_GOAHEAD +
                      data + secret + IAC + SE)
         await writer.drain()
+        await self.cache.set(
+            str(data + secret, encoding="latin-1"),
+            vm_uuid,
+            expire=CACHE_TTL,
+        )
```

Comment:
We might want to switch that around. One the GOAHEAD is out, the target-host can connect at any time. If we don't have a cache then, we lost the mapping.